examples/blog-service/README.md

```diff
@@ -86,8 +89,8 @@ Ballerina Swan Lake Update 13 (2201.13.0) will include experimental support for
     port=<DB_PORT>
     database="<DB_NAME>"
 
-    [ballerina.np.defaultModelConfig]
-    url = "<DEFAULT_MODEL_URL>"
+    [ballerina.ai.wso2ProviderConfig]
```

Comment:
```suggestion
    [ballerina.ai.wso2ModelProviderConfig]
```

wdyt?

Reply:
This comes from the ai module? https://github.com/ballerina-platform/module-ballerina-ai/blob/f56e184bc649c0af7ed02113001bd0d72925759f/ballerina/model-provider.bal#L138